client/src/document/forms/form.tsx

```diff
@@ -0,0 +1,195 @@
+import React, { useEffect, useState } from "react";
```

Comment:
most of this is just what we already had in the `edit-form.tsx`, but now that component is shared between edit & create